README.md

```diff
@@ -23,5 +28,10 @@ At run-time, `main` reads from a model configuration set in `config.py`. The con
 python3 main.py --model [regression|classification|rl]
 ```
 
+## Analysis:
+The scripts `weight_pruning.py` and `compute_ece.py` perform post-hoc analysis using saved models.
+* `weight_pruning` 1) plots the distribution of weights, 2) computes SNR of BNNs, 3) evaluates performance on pruned weights.
+* `compute_ece` 1) computes the ECE of trained model, 2) plots reliability diagram.
```

Comment:
Stupid question - what does ECE stand for?

Reply:
done